TRN-105/Table/Table.php

```diff
@@ -0,0 +1,15 @@
+<?php
+  namespace Tablearea;
+  class Table{
+    function DispTable(){
```

Comment:
Parameterised this method so that it can be generalised to show `n x n` table

Reply:
Done!